src/dns_private.tf

```diff
@@ -41,4 +41,18 @@ resource "azurerm_private_dns_a_record" "private_dns_a_record_management" {
   resource_group_name = azurerm_resource_group.rg_vnet.name
   ttl                 = 300
   records             = module.apim.*.private_ip_addresses[0]
+}
+
+# Private DNS Zone for Postgres Databases
+
+resource "azurerm_private_dns_zone" "postgres" {
+  name                = "private.postgres.database.azure.com"
+  resource_group_name = azurerm_resource_group.rg_vnet.name
+}
+
+resource "azurerm_private_dns_zone_virtual_network_link" "postgres" {
+  name                  = format("%s-postgres-private-dns-zone-link", local.project)
```

Comment:
Name should be vnet name

Reply:
I'll keep also the string `postgres` in the name, otherwise it won't be unique for different private dns zones

Reply:
For virtual network link u can use same name for different private DNS zones.

Reply:
vnet_postgres is good anyway